www/pages/index.tsx

```diff
@@ -1,49 +1,137 @@
 import styled from "@emotion/styled";
 import { GetStaticProps } from "next";
 
-import SymbolLinkTable from "@/components/Docs/SymbolLinkTable";
+import SymbolLinkTable, {
+  Sym,
+  SymbolLinkTableProps,
+} from "@/components/Docs/SymbolLinkTable";
 import Header from "@/components/Header";
 import PageLayout from "@/components/PageLayout";
+import { Search } from "@/components/core/Search";
+import {
+  flexColumn,
+  flexColumnCenter,
+  flexRow,
+} from "@/components/core/layout/helpers";
 
-import { Project, getProject, listProjects } from "@/lib/docs";
+import { Project, ProjectMetadata, getIndexMetadata } from "@/lib/docs";
+import {
+  SearchDescriptor,
+  SymbolType,
+  makeIndex,
+} from "@/lib/searchDescriptor";
 import * as url from "@/lib/url";
 
 const ProjectContainer = styled.div`
+  ${flexColumn}
   margin-top: auto;
   margin-bottom: auto;
 `;
 
-export const getStaticProps: GetStaticProps<Props> = async () => {
-  const projectNames = await listProjects();
-  const projects = await Promise.all(
-    projectNames.map(async (name) => {
-      const proj = await getProject(name);
-      return {
-        name,
-        documentation: [`${proj.metadata.summary ?? ""}`],
-      };
-    }),
+const SearchContainer = styled.div`
+  margin: ${(props) => props.theme.spacing.xl};
+  min-width: 40vw;
+`;
+
+const ProjectTables = styled.div`
+  ${flexRow}
+  @media screen and (max-width: 500px) {
```

Comment:
Could you place a TODO here to move this into the theme's (to be) breakpoints? 🙂